pkg/kube/translation/annotations/redirect.go

```diff
@@ -37,8 +41,13 @@ func (r *redirect) PluginName() string {
 func (r *redirect) Handle(e Extractor) (interface{}, error) {
 	var plugin apisixv1.RedirectConfig
 	plugin.HttpToHttps = e.GetBoolAnnotation(_httpToHttps)
+	plugin.URI = e.GetStringAnnotation(_uri)
+	retCode, err := strconv.Atoi(e.GetStringAnnotation(_retCode))
```

Comment:
We should handle the err when err != nil
If there is special logic, we should add comments that the err returned by this method is always nil